### PR TITLE
Upgrade helm version to v3.6.2

### PR DESCRIPTION
**Pull request by fralvarop**
- upgrade helm version to 3.6.2
- add --create-namespace and --wait flag to helm command to avoid terratest test failing on namespace creation when deploying a helm release.

### CODE REVIEW

File: helm/helm-release.sh.tpl [hunk 1]
```diff
@@ -46,14 +46,14 @@ if [ "$FORCE_HELM_VERSION" == "v2" ] || ( [ "$FORCE_HELM_VERSION" != "v3" ] && [
 
     {HELM_PATH} init -c
 
-    echo "{HELM_PATH} upgrade --install --tiller-namespace {TILLER_NAMESPACE} $HELM_OPTIONS --namespace {NAMESPACE} {VALUES_YAML} {RELEASE_NAME} {CHART_PATH}"
+    echo "{HELM_PATH} upgrade --install --tiller-namespace {TILLER_NAMESPACE} $HELM_OPTIONS --namespace {NAMESPACE} --wait {VALUES_YAML} {RELEASE_NAME} {CHART_PATH}"
```
Comment (danigar) [inline helm/helm-release.sh.tpl:49]: I think we should parametrize the `--wait` flag, and let it disabled by default.

Comment (danigar) [inline helm/helm-release.sh.tpl:49]: `--create_namesapace` flag is missing here. It's available in helm2 apis?

**Review by danigar — COMMENTED** [review]
Just some comments, but looks good!

File: helm/helm-release.sh.tpl (hunk 1 above)
Comment (fralvarop) [reply helm/helm-release.sh.tpl:49]: No, `--create_namesapace` it's not available in helm2 api

File: helm/helm-release.bzl [hunk 2]
```diff
@@ -102,6 +104,8 @@ helm_release = rule(
       "sops_yaml": attr.label(allow_single_file = True, mandatory = False),
       "helm_version": attr.string(mandatory = False),
       "kubernetes_context": attr.string(mandatory = False),
+      "create_namespace": attr.string(mandatory = False, default = ""),
+      "wait": attr.string(mandatory = False, default = ""),
```
Comment (ar3s3ru) [inline helm/helm-release.bzl:110]: These changes are not reflected in the README, maybe the documentation should be moved to [stardoc](https://github.com/bazelbuild/stardoc)?

**Pull request merged**